### ISSUE TITLE
EMQX container crashes : v4.0.0

### ISSUE THREAD

**Issue by vermaabhay** [emqx/emqx-rel#518]
#### Environment

- OS: Ubuntu 18.04 (host as well as docker os version)
- Erlang/OTP: 22.3
- EMQ: v4.0.0 (note: we have build emqx v4.0.0 docker on ubuntu 18.04)

#### Description
We have deployed a 3 node (4c,16g each) cluster of emqx v4.0.0 using helmcharts. I was doing a mqtt perf test and the cluster cannot accept more than 30k concurrent connections. All the 3 nodes crash beyond 30k i.e practically 10k on each. I have followed the n/w and emqx tuning guidelines as specified https://docs.emqx.io/broker/latest/en/tutorial/turn.html.

error message displayed in pod logs - /usr/bin/start.sh: line 54 : kill: emqx: arguments must be process or job IDs. This I suppose is due to the fact when the process crashed there is no log file being opened by tail command.
Even I was running in debug mode and this happened multiple time but there wasn't anything in logs.

**Comment by vermaabhay** [comment]
erts version for this image is 10.7.2, could be erl is crashing and emqx getting shutdown is just a ripple effect. Any thoughts ?

**Comment by HJianBo** [comment]
Hi @vermaabhay Maybe you can try the `start.sh` in the v4.0.7 version

**Comment by vermaabhay** [comment]
This has been fixed. Since we have built emqx image on ubuntu, the start.sh script needs to be changed a bit to get the pid of the process that keeps a tail on the previous log file in case the file gets rotated. In alpine 1st column in `ps` gives the pid while in ubuntu 2nd column  in `ps` give the pid. Therefore the pods used to crash whenever the log file rotation triggered and the start.sh script was set to exit on error.
